### PR TITLE
#61_1 add ExistsByIdsAsync

### PR DESCRIPTION
**Pull request by alesyalis**

### CODE REVIEW

File: Recrutify/Recrutify.Services/Validators/BulkUpdateStatusReasonCandidatsValidator.cs [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
                 .NotNull()
                 .NotEmpty()
                 .MaximumLength(128)
-                .When(x => x.Status == StatusDTO.Declined);
+                .When(x => x.Status == StatusDTO.Declined)
+                .WithMessage("Reason is mandatory if status is declined");
             RuleFor(x => x.Status)
                 .IsInEnum()
-                .WithMessage("Choose from existing statuses")
+                .WithMessage("Statud doesn't exist")
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/Validators/BulkUpdateStatusReasonCandidatsValidator.cs:34]: Status

File: Recrutify/Recrutify.Services/Validators/BulkCreateTestFeedbackValidator.cs [hunk 2]
```diff
@@ -23,24 +23,23 @@ public BulkCreateTestFeedbackValidator(ICandidateRepository candidateRepository)
...
+                .NotEmpty()
                 .MustAsync(CandidatesAreExistingAsync)
                 .WithMessage("One or more candidates doesn't exist");
             RuleFor(x => x.Rating)
                 .Must(r => r >= 0 && r <= 10)
                 .WithMessage("Rating is out of range");
         }
 
-        private async Task<bool> CandidatesAreExistingAsync(BulkCreateTestFeedbackDTO dto, CancellationToken cancellationToken)
+        private async Task<bool> CandidatesAreExistingAsync(BulkCreateTestFeedbackDTO dto, IEnumerable<Guid> candidatsIds, CancellationToken cancellationToken)
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/Validators/BulkCreateTestFeedbackValidator.cs:36]: candidatesIds

File: Recrutify/Recrutify.Host/Controllers/CandidatesController.cs [hunk 3]
```diff
@@ -98,7 +98,7 @@ public async Task<ActionResult> BulkCreateTestFeedbacksAsync(BulkCreateTestFeedb
             return NoContent();
         }
 
-        [Authorize(Policy = Constants.Policies.ManagerPolicy)]
+        //[Authorize(Policy = Constants.Policies.HighAccessPolicy)]
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Controllers/CandidatesController.cs:101]: Manager policy

Comment (alesyalis) [reply Recrutify/Recrutify.Host/Controllers/CandidatesController.cs:101]: i tested

File: Recrutify/Recrutify.Services/Validators/BulkCreateTestFeedbackValidator.cs (hunk 2 above)
Comment (alesyalis) [reply Recrutify/Recrutify.Services/Validators/BulkCreateTestFeedbackValidator.cs]: done

File: Recrutify/Recrutify.Services/Validators/BulkUpdateStatusReasonCandidatsValidator.cs (hunk 1 above)
Comment (alesyalis) [reply Recrutify/Recrutify.Services/Validators/BulkUpdateStatusReasonCandidatsValidator.cs]: done

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**